src/server/game/map/tile/improvement.ts

```diff
@@ -13,50 +13,85 @@ const improvementStoreCapTable: { [improvement: string]: YieldParams } = {
   'settlement': {food: 20, production: 2},
 
   'farm': {food: 20},
-}
+};
 
 const constructionCostTable: { [improvement: string]: Yield } = {
   'farm': new Yield({production: 10}),
 };
 
-export interface ImprovementData {
+const naturalImprovementTable: { [improvement: string]: boolean } = {
+  'forest': true,
+};
+
+export type ImprovementData = {
   type: string;
   pillaged: boolean;
   storage: YieldParams;
+  errand?: ErrandData;
   metadata?: any;
-}
+};
+
+export type ErrandData = {
+  storedThisTurn: YieldParams;
+  turnsToCompletion: number;
+};
 
 export class Improvement {
   type: string;
   pillaged: boolean;
   yield: Yield;
+  isNatural: boolean;
+  errand?: WorkErrand;
   metadata: any;
 
   protected traders: Trader[];
   protected suppliers: Trader[];
   protected storage: ResourceStore;
   
-  constructor(type: string, metadata?: any) {
+  constructor(type: string, baseYield: Yield, metadata?: any) {
     this.type = type;
     this.pillaged = false;
-    this.yield = improvementYieldTable[type] ?? new Yield({});
+    this.isNatural = naturalImprovementTable[type];
```

Comment:
Once again, just me being nitpicky probably, but shouldn't this be:
```suggestion
    this.isNatural = naturalImprovementTable[type] ?? false;
```